src/utils/paymentPlanStrings.tsx

```diff
@@ -35,6 +35,19 @@ export const paymentPlanInfoText = (payment: EligibilityPlanToDisplay): ReactNod
   } = payment
   const deferredDaysCount = deferred_days + deferred_months * 30
 
+  const withNoFee = () => {
+    if (
+      payment.payment_plan.every((plan) => plan.customer_fee === 0 && plan.customer_interest === 0)
+    ) {
+      return (
+        <>
+          {' '}
```

Comment:
Do we really need this line ?

Reply:
I'm not a big fan either of `{' '}`, 
we could remove this and add some `css` to manage spacing, but if we just remove it, there won't be any space between the first text, `2 x 225,00 €` and `(sans frais)` so it'll appear like `2 x 225,00 €(sans frais)` :/